modules/claims_api/app/swagger/claims_api/v2/production/swagger.json

```diff
@@ -849,7 +850,6 @@
                     },
                     "evidenceWaiverSubmitted5103": {
                       "type": "boolean",
-                      "nullable": true,
```

Comment:
I was curious if we should set these to false, but according to this, we only need `nullable: true` if it could potentially be null? I feel like this means we're safe to remove this. 

The very bottom of this: https://swagger.io/docs/specification/data-models/data-types/